src/desktop/src/ui/components/input/Seed.js

```diff
@@ -123,6 +123,13 @@ class SeedInput extends React.PureComponent {
         });
     };
 
+    onTextDrop = async (seed) => {
```

Comment:
Is the `async` here necessary? It doesn't look like `await` is used in this function